### PR TITLE
New buttons components

### PR DESCRIPTION
**Pull request by felipesaev**
**CHANGELOG** :memo:

* create new buttons

### CODE REVIEW

File: src/yellow/newcomponets/buttons/_index.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  border-image: initial;
+  border-radius: $size-border-radius-m;
+  border-style: $border-style-default;
+  border-width: $size-border-width-thick;
+  color: $color-interface-action-light;
+  cursor: pointer;
+  display: flex;
+  font-size: $size-font-s;
+  justify-content: center;
+  margin-bottom: 10px;
```
Comment (TrevelinT) [inline src/yellow/newcomponets/buttons/_index.scss:17]: Button needs to have margin?

File: src/yellow/newcomponets/buttons/_index.scss [hunk 2]
```diff
@@ -0,0 +1,122 @@
+@import '../../../../node_modules/@getninjas/niten-tokens/build/web/niten-tokens';
+
+.button {
+  cursor: pointer;
+  font-size: $size-font-s;
+  width: 100%;
+  display: flex;
+  -webkit-box-align: center;
```
[the remark this answers is not in the document]
Comment (costalince) [reply src/yellow/newcomponets/buttons/_index.scss:4]: esses que tem _-webkit_ pode remover

File: src/yellow/newcomponets/buttons/_index.scss [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  align-items: center;
+  -webkit-box-pack: center;
```
[the remark this answers is not in the document]
Comment (costalince) [reply src/yellow/newcomponets/buttons/_index.scss:5]: esses que tem _-webkit_ pode remover

File: src/yellow/newcomponets/buttons/_index.scss (hunk 1 above)
Comment (felipesaev) [reply src/yellow/newcomponets/buttons/_index.scss]: no, removed. Thanks

**Pull request merged**